.circleci/config.yml

```diff
@@ -64,4 +64,3 @@ workflows:
                 - master
                 - next
```

Comment:
We want to keep `next` and `next-major` around?

Reply:
Yep, they were there before we started to work on `experimental`, so it's fair to leave them here